requirements-notebooks.txt

```diff
@@ -1,2 +1,4 @@
+Jinja2
```

Comment:
I presume this is used for the reports but I don't see Jinja2 being imported in the notebook itself. I'm also not seeing the Jinja2 template.

Reply:
Tests error unless Jinja2 is added to requirements.